docs/contributing/coding-style.md

```diff
@@ -776,6 +778,34 @@ tests := map[string]func(t *testing.T) test {
     }
     ```
 
+#### Input agreement
+
+1. test case name
+
+    Test case name should be readable and meaningful for understanding each test easily.
+    If you create a new test, the test name should be named based on the below-naming templates.
+
+    - Success:
+        - Start with `success` or `{verb} success` or `success {verb}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `Set success when value is default value`
+    - Fail:
+        - Start with `fail` or `{verb} fail` or `fail {verb}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `fail option setting when value is invalid value(string)`
+    - Return:
+        - When the test will not much `Success` or `Fails`, please use `Return` pattern.
+        - Start with `Returns {Object}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `Returns invalid error when the input is invalid`
+
+1. input values
+
+    Input value for testing should be a meaningful value.
+    It is because we should test with more realistic value as user use.
+
+    For example, when you use `host` for test value, you should set `your hostname (e.g. vald.vdaas.com)` as `host`.
+
```

Comment:
should we include a link to the unit test guideline here?

Reply:
it's good idea 😎

Reply:
But, I cannot image where we can set link 🤔

Reply:
maybe we can move all the contents of test section to the new doc can include only the like here.
https://github.com/vdaas/vald/blob/064c7fb5035596928a5696a7dc93eec81cb24856/docs/contributing/coding-style.md#test

Reply:
I think it'd be better to separate it.
cuz, this document only targets unit test, but coding-guideline targets all test (unit, integrate, api...)